addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/SubResourceIntegrityAttributeScanner.java

```diff
@@ -124,6 +125,20 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
         };
     }
 
+    private static Predicate<Element> htmlNotInline() {
+        // 'else' branch won't ever execute since everything evaluated is a SCRIPT/LINK.
+        // Leaving it in place for clarity.
+        return element -> {
+            if (element.getName() == HTMLElementName.SCRIPT) {
+                return !element.getAttributeValue("src").startsWith("data:");
+            } else if (element.getName() == HTMLElementName.LINK) {
+                return !element.getAttributeValue("href").startsWith("data:");
+            } else {
+                return true;
+            }
+        }
```

Comment:
The build is failing because of a missing semi-colon here.

```
> Task :addOns:pscanrulesAlpha:compileJava FAILED
/home/travis/build/zaproxy/zap-extensions/addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/SubResourceIntegrityAttributeScanner.java:139: error: ';' expected
        }
         ^
1 error
```

Reply:
On it, thanks.